Patches/Hive Armory/HiveArmory_Weapons.xml

```diff
@@ -0,0 +1,121 @@
+                        <li Class="CombatExtended.ToolCE">
+                            <label>shaft</label>
+                            <capacities>
+                            <li>Blunt</li>
+                            </capacities>
+                            <power>6</power>
+                            <cooldownTime>1.31</cooldownTime>
+                            <armorPenetrationBlunt>2.025</armorPenetrationBlunt>
+                            <linkedBodyPartsGroup>Shaft</linkedBodyPartsGroup>
+                        </li>
+                        <li Class="CombatExtended.ToolCE">
+                            <label>edge</label>
+                            <capacities>
+                            <li>Cut</li>
+                            </capacities>
+                            <power>36</power>
+                            <cooldownTime>2.0</cooldownTime>
+                            <chanceFactor>1.165</chanceFactor>
+                            <armorPenetrationBlunt>5.0</armorPenetrationBlunt>
+                            <armorPenetrationSharp>0.9</armorPenetrationSharp>
+                            <linkedBodyPartsGroup>Edge</linkedBodyPartsGroup>
+                        </li>
+                        <li Class="CombatExtended.ToolCE">
+                            <label>point</label>
+                            <capacities>
+                            <li>Stab</li>
+                            </capacities>
+                            <power>27</power>
+                            <cooldownTime>1.68</cooldownTime>
+                            <chanceFactor>1.165</chanceFactor>
+                            <armorPenetrationBlunt>2.75</armorPenetrationBlunt>
+                            <armorPenetrationSharp>1.5</armorPenetrationSharp>
+                            <linkedBodyPartsGroup>Point</linkedBodyPartsGroup>
+                        </li>
+                    </tools>
+                </value>
+            </li>
+
+            <li Class="PatchOperationAdd">
+                <xpath>Defs/ThingDef[defName="Kenshi_Polearm"]/statBases</xpath>
+                <value>
+                    <Bulk>11</Bulk>
+                    <MeleeCounterParryBonus>1.33</MeleeCounterParryBonus>
+                </value>
+            </li>
+
+            <li Class="PatchOperationReplace">
+                <xpath>Defs/ThingDef[defName="Kenshi_Polearm"]/equippedStatOffsets</xpath>
+                <value>
+	<equippedStatOffsets>
+                    <MeleeCritChance>0.27</MeleeCritChance>
+                    <MeleeParryChance>1.1</MeleeParryChance>
+                    <MeleeDodgeChance>0.8</MeleeDodgeChance>
+	</equippedStatOffsets>
+                </value>
+            </li>
+
+            <li Class="PatchOperationReplace">
+                <xpath>Defs/ThingDef[defName="Kenshi_Staff"]/tools</xpath>
+                <value>
+                    <tools>
+                        <li Class="CombatExtended.ToolCE">

```

Comment:
I'd give one attack the blunt capacity and one the poke capacity, rather than having one of both. Additionally, the second melee attack is way too strong--a balanced quarterstaff is going to be fast, but won't have the weight to deliver a truly heavy blow.

I've attached a screenshot with my recommendation for the two attack stats. They're calculated from the spreadsheet, modeling a roughly 1.2" diameter, 6' long quarterstaff made of steel.

![Staff](https://user-images.githubusercontent.com/38633594/121999628-8a4e7080-cd7b-11eb-8092-d9f613de6822.PNG)

Reply:
These stats are absurdly bad for their cost, they are two-handed, full metal staffs so I'm trying to make them slightly better than one-handed mace. These stats are worse than a club, which has an extremely long melee cooldown for no particular reason.

Reply:
I've nerfed the staff alittle and intend to tweak the cost of it at a later date.
I intend to keep the poke capacity on both attack since the staff is identical on both end+it is a way to differentiate it from other blunt weapons.